src/applications/_mock-form-ae-design-patterns/tests/e2e/pattern1/taskGreen.cypress.spec.js

```diff
@@ -4,7 +4,7 @@ import mockPrefills from '../../../mocks/endpoints/in-progress-forms/mock-form-a
 
 describe('Prefill pattern - Green Task', () => {
   beforeEach(() => {
-    cy.login(mockUsers.loa3User);
+    cy.intercept('GET', '/v0/user', mockUsers.loa3User).as('mockUser');
```

Comment:
`cy.login` sets `hasSession` to true in localStorage, this test was relying on the user initialization happening slowly to work.